components/dialog/predefined.md

```diff
@@ -21,14 +24,28 @@ To use these dialogs (popup messages), receive a cascading parameter of type `Te
 public DialogFactory Dialogs { get; set; }
 ````
 
-The predefined dialogs can only be used inside child Razor components of the [`TelerikRootComponent`]({%slug rootcomponent-overview%}). This is easily achieved in standard Blazor apps where the `TelerikRootComponent` resides inside the layout file. When [using UI for Blazor in ASP.NET apps]({%slug telerik-blazor-in-asp-net-app%}), an additional child component is needed.
-
 There are three available ready-made dialogs:
 
 * [Alert](#alert)
 * [Confirm](#confirm)
 * [Prompt](#prompt)
 
+### TelerikRootComponent Dependency
+
+The `DialogFactory` cascading parameter can only be consumed inside child Razor components of the [`TelerikRootComponent`]({%slug rootcomponent-overview%}#purpose). This is easily achieved in standard Blazor apps where the `TelerikRootComponent` resides inside the layout file.
+
+* To show predefined dialogs in `MainLayout.razor`, [add the `TelerikRootComponent` in a parent layout]({%slug rootcomponent-overview%}#using-telerikrootcomponent).
+* When [using UI for Blazor in ASP.NET apps]({%slug telerik-blazor-in-asp-net-app%}), an additional child component is needed.
+
+### Using `await`
```

Comment:
Don't use monospace in the headings to avoid overloading the reader with too many styles.
```suggestion
### Using await
```